### ISSUE TITLE
(plugin typescript) TypeError: _a.close is not a function

### ISSUE THREAD

**Issue by CSchulz** [rollup/plugins#250]
- Rollup Plugin Name: typescript
- Rollup Plugin Version: 4.0.0
- Rollup Version: 1.32.0
- Operating System (or Browser):
- Node Version: 10.16.0

### How Do We Reproduce?

https://repl.it/repls/CriminalQuestionableAbilities

### Expected Behavior
Should work and bundle all.

### Actual Behavior
Fails:

```
npx rollup -c rollup.config.ts

src/index.ts → dist/index.js, dist/index.es5.js...
[!] (plugin typescript) TypeError: _a.close is not a function
TypeError: _a.close is not a function
    at Object.buildEnd (/home/runner/CriminalQuestionableAbilities/node_modules/@rollup/plugin-typescript/dist/index.js:482:72)
    at Promise.resolve.then (/home/runner/CriminalQuestionableAbilities/node_modules/rollup/dist/rollup.js:12899:25)

exit status 1
```

**Comment by NotWoods** [comment]
`typescript` needs to be included as a dependency to use the plugin. Adding it to the package.json in the REPL fixes the issue.

**Comment by maslade** [comment]
For anyone else stuck on this -- I encountered the same error message in my project that I was able to fix by upgrading Typescript from 3.3 to 3.8:

```
-    "typescript": "3.3.3333",
+    "typescript": "^3.8.3",
```

My project has a lot of other stuff in it so I'm not confident this was "the issue", but might help someone. I notice the peer dependencies for plugin-typescript only ask for Typescript >= 2.1.
